### ISSUE TITLE
Https

### ISSUE THREAD

**Issue by Theoldnation** [polymorphicshade/NewPipe#259]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Does this fork uses https connections to svs to do anything? I had seen normal newpipe already does that but this one apparently doesn't and that could be seen looking at log error reports and in analysis via virustotal comparing original newpipe and this one

**Comment by Atemu** [comment]
What exactly is "svs"?

**Comment by polymorphicshade** [comment]
@Theoldnation This fork connects in the exact same way vanilla NewPipe does.
Can you post an example of this fork not using HTTPS? I ask not because I doubt you, but because I did not change anything related to using HTTP vs HTTPS/

**Comment by Theoldnation** [comment]
Yup, I've also verified it checking other things, I will close the matter right now, thanks for solving my questions
